apps/emqx/src/emqx_release.erl

```diff
@@ -25,70 +25,47 @@
     version_with_prefix/0,
     vsn_compare/1,
     vsn_compare/2,
-    get_flavor/0,
-    on_load/0
+    get_flavor/0
 ]).
 
 -ifdef(TEST).
 -export([set_flavor/1]).
 -endif.
 
--on_load(on_load/0).
-
 -include("emqx_release.hrl").
 
+%% TODO: delete
 -ifndef(EMQX_RELEASE_EDITION).
--define(EMQX_RELEASE_EDITION, ce).
+-define(EMQX_RELEASE_EDITION, ee).
 -endif.
 
--define(EMQX_DESCS, #{
-    ee =>
-        case get_flavor() of
-            official -> "EMQX Enterprise";
-            Flavor -> io_lib:format("EMQX Enterprise(~s)", [Flavor])
-        end,
-    ce => "EMQX"
-}).
-
--define(EMQX_REL_NAME, #{
-    ee => <<"Enterprise">>,
-    ce => <<"Opensource">>
-}).
-
--define(EMQX_REL_VSNS, #{
-    ee => ?EMQX_RELEASE_EE,
-    ce => ?EMQX_RELEASE_CE
-}).
-
--define(EMQX_REL_VSN_PREFIX, #{
-    ee => "e",
-    ce => "v"
-}).
-
-%% @hidden Initialize edition. Almost static. use persistent_term to trick compiler.
--spec on_load() -> ok.
-on_load() ->
-    persistent_term:put('EMQX_RELEASE_EDITION', ?EMQX_RELEASE_EDITION).
+-define(EMQX_DESCS,
+    case get_flavor() of
+        official -> "EMQX Enterprise";
+        Flavor -> io_lib:format("EMQX Enterprise(~s)", [Flavor])
```

Comment:
In `rebar.config.erl`, these are renamed to "EMQX Platform".  Should it be the same here?